src/app/public/modules/file-attachment/file-attachment.component.scss

```diff
@@ -54,3 +54,7 @@
   box-shadow: 0 0 5px #666;
   margin: 10px;
 }
+
+.sky-file-attachment-drop-description {
+  display: none;
```

Comment:
Aren't items with `display: none` usually not read by screen readers? Should this be styled differently? See: https://webaim.org/techniques/css/invisiblecontent/